hazelcast/protocol/codec/count_down_latch__await_codec.py

```diff
@@ -0,0 +1,29 @@
+from hazelcast.serialization.bits import *
```

Comment:
Let's delete this codec again :smile: 
I haven't fixed the client protocol code generator yet, so this is generated unnecessarily